java/src/jmri/jmrit/logixng/actions/swing/ActionListenOnBeansTableSwing.java

```diff
@@ -120,6 +97,35 @@ protected void createPanel(@CheckForNull Base object, @Nonnull JPanel buttonPane
         }
     }
 
+    private void comboListener( ActionListenOnBeansTable action, JComboBox<TableRowOrColumn> rowOrColBox,
+            JComboBox<String> rowOrColnameComboBox, BeanSelectPanel<NamedTable> tblbeanPanel ) {
```

Comment:
I would appreciate if you could rename `tblbeanPanel` to `tableBeanPanel`. I'm not a native English speaker and it makes the variable easier to read.